.gitignore

```diff
@@ -30,3 +30,6 @@ __pycache__
 
 # Log files
 *.log
+coverage-report
```

Comment:
Don't worry too much about the project structure. Definitely make notes but consider not blocking this PR on dir structure b/c I'm going to refactor in later tasks. The meat and potatoes of this PR starts at `services/discovery/src/app.py`.